firebase-firestore-mixin.html

```diff
@@ -205,15 +204,14 @@
         this._firestoreProps[name] = config;
 
         const args = config.props.concat(config.observes);
+        this._firestoreUpdateBinding(name, ...args.map(x => this[x]));
```

Comment:
Thought more about the issue at hand here, and it wasn't so much that I didn't want this called in every `_firestoreBind` call, but that I didn't want it called after the `_createMethodObserver` had been created because it meant that when `_firestoreUnlisten` was called as part of the initial `_firestoreUpdateBinding` call would immediately trigger a second call to `_firestoreUpdateBinding` because of the zeroing out of the value for the property being bound to at line 251.

Reply:
Take a look at how the observer is created: https://github.com/firebase/polymerfire/pull/327/files#diff-ed26b76bc80e40b84633288e109db714R211

The `name` is passed in quotes, just to indicate for which property arguments have changed. It does not observe the property itself.

Reply:
Indeed. I didn't follow that through line deep enough.

With the code ordered this way, the initial properties flush of the `args` is able to be passed before `_firestoreUnlisten` makes the call to `setProperties`, which is why it's not called twice at runtime. This is due to the initial call to `_flushProperties ` not occurring till https://github.com/Polymer/polymer/blob/master/lib/mixins/property-effects.html#L1661

Not sure if there is a more idiomatic way of achieving the timing we need here.